Brutus/Project.toml

```diff
@@ -3,5 +3,9 @@ uuid = "44ccd279-0a44-4492-af09-0e34b2907bcc"
 authors = ["Valentin Churavy <v.churavy@gmail.com>", "Leon Shen <leon@yhls.org>"]
 version = "0.1.0"
 
+[deps]
+LLVM = "929cbde3-209d-540e-8aea-75f648917ca0"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
should probably not be a dependency, but should be in the extras section.

Reply:
According to https://julialang.github.io/Pkg.jl/v1/creating-packages/#Test-specific-dependencies-1 I should add it to Brutus/test/Project.toml? I just updated this branch to do this.